### PR TITLE
Refactor the GServer - Part 2

### PR DESCRIPTION
**Pull request by LoneBoco**
Large refactor project on the gserver to consolidate style and better organize code.

- [x] Develop a coding style to use with the project.
- [x] Apply the code style to all .h/.cpp files.
- [x] Rename all classes and structs to remove prefixes.
- [x] Rename all private/protected member variables of classes for consistency.
- [x] Rename all functions of classes for consistency.

### CODE REVIEW

File: server/include/Player.h [hunk 1]
```diff
@@ -54,391 +55,393 @@ struct ShootPacketNew {
 
 struct SCachedLevel
```
Comment (xx-shitai-xx) [inline server/include/Player.h:56]: Shouldn't this be renamed to `CachedLevel` ?

File: server/include/Scripting/GS2ScriptManager.h [hunk 2]
```diff
@@ -31,18 +29,18 @@ class GS2ScriptManager
...
 	// Async Compile
-	void queueCompileJob(const std::string &script, user_callback_type& finishedCb);
+	void queueCompileJob(const std::string& script, user_callback_type& finishedCb);
 
 	// Sync Compile
 	GS2Context _context;
 	void syncCompileJob(const std::string& script, user_callback_type& finishedCb);
-	
+
 	//
```
Comment (xx-shitai-xx) [inline server/include/Scripting/GS2ScriptManager.h:38]: Empty comment
```suggestion
```

**Review by xx-shitai-xx — COMMENTED** [review]
Will continue review tomorrow

File: server/include/Player.h (hunk 1 above)
Comment (LoneBoco) [reply server/include/Player.h:56]: Yep.  It was missed.

File: server/include/Scripting/GS2ScriptManager.h (hunk 2 above)
Comment (LoneBoco) [reply server/include/Scripting/GS2ScriptManager.h]: Fixed.

**Review by xx-shitai-xx — APPROVED** [review]
Looks good to me! :)

**Pull request merged**